driver-core/src/main/com/mongodb/operation/MixedBulkWriteOperation.java

```diff
@@ -291,7 +291,15 @@ private BulkWriteResult executeBulkWriteBatch(final WriteBinding binding, final
             }
             throw exception;
         } else {
-            return retryExecuteBatches(binding, currentBatch, exception);
+            if (!(exception instanceof MongoWriteConcernWithResponseException)
```

Comment:
This wasn't reverted properly.  Looks like the conditional is duplicated instead.

Reply:
Fixed the revert.